Translated/Dialog/Shioji/ssn_015510.csv

```diff
@@ -0,0 +1,39 @@
+ssn_015510_0001#0,"""Yo! Takin' good care of your equipment?"""
+ssn_015510_0002#0,"""It don't matter how strong an ARKS is; if their equipment's in tatters, they can't fight at their best."""
+ssn_015510_0003#0,"""Think of your gear as your most valuable partner, and afford it as much respect."""
+ssn_015510_0101#0,"""Just the ARKS I wanted to see."""
+ssn_015510_0102#0,"""I'm Shioji, and I do equipment research here in Stia."""
+ssn_015510_0103#0,"""What say you sit a spell and listen to some advice I've got for ya?"""
+ssn_015510_0201#0,"""I need you to get some data on the Ore known as \""Hexakite\""."""
+ssn_015510_0202#0,"""They say it can be found in Outer Mediola: Sector 2. Thanks again."""
+ssn_015510_0301#0,"""Oh, you're busy. Let me know if you get some spare time."""
+ssn_015510_0401#0,"""You're back! Alright, lay 'em out here and I'll start scanning them."""
+ssn_015510_0402#0,"""...Hm-hm... I see."""
+ssn_015510_0403#0,"""As I thought, this Ore structure is quite different from those found in Aelio."""
+ssn_015510_0404#0,"""We might be able to use these to make adjustments to new equipment as well."""
+ssn_015510_0405#0,"""I'll be sure to inform the technology department and have them use this information to improve our equipment."""
+ssn_015510_0406#0,"""It looks like Stia may have its own wealth of useful Ores."""
+ssn_015510_0407#0,"""Alright! The researcher in me is starting to get excited!"""
+ssn_015510_0408#0,"""I got some valuable data thanks to you."""
+ssn_015510_0409#0,"""I appreciate it. I hope to be able to depend on you if something else comes up."""
+ssn_015510_1001#0,"""Here in Stia, every day is a battle."""
+ssn_015510_1002#0,"""If your gear ain't in perfect condition, it doesn't matter how many lives you've got, they won't be enough."""
+ssn_015510_1101#0,"""Ho there! I heard you played a big part in that last battle."""
+ssn_015510_1102#0,"""Taking those abilities into account, there's another favor I'd like to ask of you."""
+ssn_015510_1103#0,"""Would you consider it?"""
+ssn_015510_1201#0,"""\""Inferium\"" can be gathered in Outer Mediola: Sector 1."""
+ssn_015510_1202#0,"""There's rumours that you can only find it during certain weather."""
+ssn_015510_1203#0,"""I know it's a lot to ask, but since it's you, I have faith that it'll work out."""
+ssn_015510_1301#0,"""Oh, alright... Come back if you change your mind!"""
+ssn_015510_1401#0,"""Oh, you're back. Let me start the data scan right away."""
+ssn_015510_1402#0,"""...Hum. So this is \""Inferium\"". This Ore is even more impressive than rumoured!"""
+ssn_015510_1403#0,"""It's strong, and yet it'd be easy to process."""
+ssn_015510_1404#0,"""...Ahh, just looking at it, all sorts of possibilities come to mind."""
+ssn_015510_1405#0,"""We could make some incredible things with this, there's no doubt about that."""
+ssn_015510_1406#0,"""It's been a while since the researcher in me was this fired up! I look forward to the coming days!"""
```

Comment:
This last bit could do with rewording.

Reply:
get ur head out of the gutter

Reply:
Didn't even occur to me. I just think it reads a bit unnatural